package.json

```diff
@@ -31,26 +32,27 @@
   },
   "license": "MIT",
   "dependencies": {
-    "contacts-pane": "^2.6.0",
+    "contacts-pane": "2.6.0-509a1d91",
     "lit-html": "^2.0.2",
-    "pane-registry": "^2.4.6",
+    "pane-registry": "2.4.6-7ecb64b0",
     "rdflib": "^2.2.17",
-    "solid-logic": "^1.3.13",
-    "solid-ui": "^2.4.15",
+    "solid-logic": "1.3.13-60e05a84",
+    "solid-ui": "2.4.16-74e478b4",
     "validate-color": "^2.2.1"
   },
   "devDependencies": {
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.0",
     "@babel/preset-typescript": "^7.16.0",
+    "@nestjs/cli": "^8.2.0",
```

Comment:
**question** what are we using that for?

Reply:
It used to be pulled in by some other lib and it is not anymore... related to webpack: https://stackoverflow.com/questions/66809411/webpack-typescript-errors-on-fresh-nestjs-app

Reply:
That's weird, since we are not using NestJS at all, I think this should be investigated further

Reply:
I will resolve this conversation for now to be able to move towards closing the PR. We should take a close look at all our dependencies of all modules from time to time, agree.